bin/mink-zombie-server.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/node
+var net = require('net');
+var zombie = require('zombie');
+var Tough = require('zombie/node_modules/tough-cookie');
+var browser = null;
+var pointers = [];
+var buffer = '';
+var host = process.env.HOST || '127.0.0.1';
```

Comment:
well, I used the same default than in the Server constructor. The idea is to be able to run the node script manually without having to set the env variables. Not using a default port is not common.